### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by frodare**

### CODE REVIEW

File: src/components/Manifest.js [hunk 1]
```diff
@@ -25,23 +26,32 @@ const StandardManifest = props => {
...
+          <PageSizer className='row-limit form form-control' pageSizes={pageSizes} pageSizeLabelGenerator={pageSizeLabelGenerator} />
+        </Col>
+        <Col className='text-center'>
+          <DefaultControlsStatus statusMessageGenerator={statusMessageGenerator} />
+        </Col>
+        <Col>
+          <Pager />
+        </Col>
+      </Row>
+      <Debug />
```
Comment (zeriley) [inline src/components/Manifest.js]: Debug

File: src/index.js [hunk 2]
```diff
@@ -1,29 +1,7 @@
...
-import { ASCENDING, DESCENDING, NOT_SORTED } from './constants/sortDirections'
+import Filter from './components/Filter'
 
 export {
-  useManifest,
-  useCell,
-  useHeaderCell,
-  usePager,
   Manifest,
-  Debug,
```
Comment (zeriley) [inline src/index.js]: Might want to continue to expose the debug as a passthrough to use-manifest's Debug

File: package.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "add123-manifest",
-  "version": "1.1.6",
+  "version": "0.0.1",
```
[the remark this answers is not in the document]
Comment (frodare) [reply package.json]: that was from use-manifest, will set it to 1.0.0

File: src/index.js (hunk 2 above)
Comment (frodare) [reply src/index.js]: I added a debug prop to handle this.  I am not sure it would be easy to insert Debug otherwise

**Pull request merged**